docs/style.css

```diff
@@ -36,12 +50,33 @@
   color: var(--theme-foreground-faint);
 }
 
-.observablehq-link-active a[target=_blank]:not(:hover, :focus)::after {
+.observablehq-link-active a[target="_blank"]:not(:hover, :focus)::after {
   color: var(--theme-foreground-focus);
 }
 
 #observablehq-main {
   min-height: calc(100vh - 23.8rem);
+  font-size: 16px;
+  font-weight: 500;
```

Comment:
These should probably be on the body instead of `#observablehq-main`.

Reply:
I started out like this, but body also affects the other elements on the page, like the left nav and the ToC. Those are overwritten by more specific css rules to get them back to only `var(--sans-serif)` so it probably doesn't matter, but I think `#observable-main` was the intended scope of the font change.

Reply:
Changing it back to only be for `#observablehq-main`